base/inc/fields/number.class.php

```diff
@@ -28,13 +45,12 @@ protected function get_default_options() {
 	}
 
 	protected function get_input_attributes() {
-		if ( empty( $this->step ) ) {
-			return array();
-		}
-	
-		return array(
+		$input_attributes = [
```

Comment:
Can you please switch this to a "long" array? For newly introduced code we try to maintain WordPress PHP Coding Standards.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays